roles/edpm_telemetry/templates/node_exporter.json.j2

```diff
@@ -39,8 +39,7 @@
     "volumes": [
 {% if tls_cert_exists|bool %}
         "{{ edpm_telemetry_config_dest }}/node_exporter.yaml:/etc/node_exporter/node_exporter.yaml:z",
-        "{{ edpm_telemetry_certs }}:/etc/node_exporter/tls:z",
+        "{{ edpm_telemetry_certs }}:/etc/node_exporter/tls:z"
 {% endif %}
-        "/var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket:rw,z"
```

Comment:
Why was it added at the first place?

Reply:
I believe it is for this options

"--collector.systemd",
"--collector.systemd.unit-include=(edpm_.*|ovs.*|openvswitch|virt.*|rsyslog)\\.service",

to work. I think this is not needed anymore as it has been replaced by the heartbeat service.

Reply:
Unfortunately it is not superseded, because we don't have all the services on compute nodes containerized any more. We gonna need to get the mount back and enable systemd collector, so that we cover all services' status. I'm gonna investigate to make it work as in the past before the change with selinux labels happened.